packages/tokens/data/icons.json

```diff
@@ -0,0 +1,763 @@
+{
```

Comment:
Should the files in `data` be commited?

Reply:
I decided to commit them, this way we can build tokens without fetching which simplifies the logic when fetching just a specific library.